addons/website_sale/models/sale_order.py

```diff
@@ -92,7 +94,12 @@ def _cart_update(self, cr, uid, ids, product_id=None, line_id=None, add_qty=0, s
             if not line_id:
                 values = self._website_product_id_change(cr, uid, ids, so.id, product_id, qty=1, context=context)
                 line_id = sol.create(cr, SUPERUSER_ID, values, context=context)
-                sol._compute_tax_id(cr, SUPERUSER_ID, [line_id], context=context)
+                try:
```

Comment:
Why doing it here ? If there is an error it should be managed in the taxcloud code. Not making a try-catch if one I want to install the module.